bin/dktl

```diff
@@ -47,7 +47,9 @@ fi
 export DKTL_PROJECT_DIRECTORY
 
 DKTL_DIRECTORY=$(which dktl)
-DKTL_DIRECTORY=$(readlink $DKTL_DIRECTORY)
+if [[ -L $(which dktl) ]]; then
```

Comment:
This was added because on Jenkins, we've been just adding the dktl bin dir directly to the path (also renaming dktl.sh to just dktl) so in some cases dktl is not a symlink.